### ISSUE TITLE
Zoom parameter not changing zoom level?

### ISSUE THREAD

**Issue by breadadams** [SubZane/simplegmaps#15]
Hey! It seems the zoom parameter isn't actually changing the zoom level of my map, here's the js I'm using:

    $('#contactMap').simplegmaps({
		MapOptions: {
			zoom: 4,
			panControl: false,
			scrollwheel: false,
			styles: [{"featureType":"poi","elementType":"all","stylers":[{"hue":"#000000"},{"saturation":-100},{"lightness":-100},{"visibility":"off"}]},{"featureType":"poi","elementType":"all","stylers":[{"hue":"#000000"},{"saturation":-100},{"lightness":-100},{"visibility":"off"}]},{"featureType":"administrative","elementType":"all","stylers":[{"hue":"#000000"},{"saturation":0},{"lightness":-100},{"visibility":"off"}]},{"featureType":"road","elementType":"labels","stylers":[{"hue":"#ffffff"},{"saturation":-100},{"lightness":100},{"visibility":"off"}]},{"featureType":"water","elementType":"labels","stylers":[{"hue":"#000000"},{"saturation":-100},{"lightness":-100},{"visibility":"off"}]},{"featureType":"road.local","elementType":"all","stylers":[{"hue":"#ffffff"},{"saturation":-100},{"lightness":100},{"visibility":"on"}]},{"featureType":"water","elementType":"geometry","stylers":[{"hue":"#ffffff"},{"saturation":-100},{"lightness":100},{"visibility":"on"}]},{"featureType":"transit","elementType":"labels","stylers":[{"hue":"#000000"},{"saturation":0},{"lightness":-100},{"visibility":"off"}]},{"featureType":"landscape","elementType":"labels","stylers":[{"hue":"#000000"},{"saturation":-100},{"lightness":-100},{"visibility":"off"}]},{"featureType":"road","elementType":"geometry","stylers":[{"hue":"#bbbbbb"},{"saturation":-100},{"lightness":26},{"visibility":"on"}]},{"featureType":"landscape","elementType":"geometry","stylers":[{"hue":"#dddddd"},{"saturation":-100},{"lightness":-3},{"visibility":"on"}]}]
		}
	});

Is there anything blatantly obvious that I'm missing? This is driving me mad!

Cheers

**Comment by SubZane** [comment]
This has to do with the map using zoom-to-fit-bounds that auto zooms so that all markers will fit.

I've added an option to disable this so that you can set your own zoom level. Check out version 1.1.0 and the docs :)

**Comment by breadadams** [comment]
Awesome! However... I've set the `ZoomToFitBounds` to false *and* added `data-center="true"` to the marker element and *still* can't change the zoom level. I'm doing it right, yeah?

```
$('#contactMap').simplegmaps({
	ZoomToFitBounds: false,
	MapOptions: {
		zoom: 10,
		panControl: false,
		scrollwheel: false,
		styles: [{"featureType":"water","elementType":"geometry","stylers":[{"color":"#ededed"},{"lightness":17}]},{"featureType":"landscape","elementType":"geometry","stylers":[{"color":"#F3F3F3"},{"lightness":20}]},{"featureType":"road.highway","elementType":"geometry.fill","stylers":[{"color":"#ffffff"},{"lightness":17}]},{"featureType":"road.highway","elementType":"geometry.stroke","stylers":[{"color":"#ffffff"},{"lightness":29},{"weight":0.2}]},{"featureType":"road.arterial","elementType":"geometry","stylers":[{"color":"#ffffff"},{"lightness":18}]},{"featureType":"road.local","elementType":"geometry","stylers":[{"visibility":"on"},{"color":"#ffffff"},{"lightness":16}]},{"featureType":"poi","elementType":"geometry","stylers":[{"color":"#f5f5f5"},{"lightness":21}]},{"featureType":"poi.park","elementType":"geometry","stylers":[{"color":"#e8e8e8"},{"lightness":21}]},{"elementType":"labels.text.stroke","stylers":[{"visibility":"off"},{"color":"#ffffff"},{"lightness":16}]},{"elementType":"labels.text.fill","stylers":[{"saturation":36},{"color":"#9e9ea0"},{"lightness":0}]},{"elementType":"labels.icon","stylers":[{"visibility":"off"}]},{"featureType":"transit","elementType":"geometry","stylers":[{"color":"#dbdbdb"},{"lightness":19}]},{"featureType":"administrative","elementType":"geometry.fill","stylers":[{"color":"#fefefe"},{"lightness":20}]},{"featureType":"administrative","elementType":"geometry.stroke","stylers":[{"color":"#fefefe"},{"lightness":17},{"weight":1.2}]}]
	}
});
```

**Comment by SubZane** [comment]
hmm weird. The online example works

Look at the example (click Markers->Markers with custom zoom level)
http://www.andreasnorman.com/simplegmaps/

**Comment by LouTerrailloune** [comment]
I have the same problem. This is because map.fitBounds() is called even if ZoomToFitBounds is false (round line 223). Ping @SubZane .

**Comment by SubZane** [comment]
Just now fixed in [1.1.2](https://github.com/SubZane/simplegmaps/tree/1.1.2)
